src/pages/ChooseProfile/ProfileRoleContainer/index.tsx

```diff
@@ -3,11 +3,13 @@ import ProfileRole from "./ProfileRole";
 import { StyledButton, StyledProfileRoleContainer } from "./styles";
 import rolesData from "./rolesData.json";
 import { IRoleData, ISelectedRol } from "./types";
+import { useTranslation } from "react-i18next";
 
 const ProfileRoleContainer: React.FC = (): JSX.Element => {
     const Data: IRoleData[] = rolesData;
     const [selected, setSelected] = useState<ISelectedRol | null>(null);
     const [buttonDisabled, setButtonDisabled] = useState(true);
+    const { t } = useTranslation();
```

Comment:
Solo una sugerencia, pero bien! 😄 
```suggestion
    const { t } = useTranslation("ProfileRole");
```